websites/A/Animevost/presence.ts

```diff
@@ -0,0 +1,70 @@
+const presence = new Presence({
+		clientId: "978634361266110534"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+function GetSeria() {
+	const all: string[] = [],
+		nodes = document.querySelectorAll("div[id='items']>div");
+	for (let i = 0; nodes[i]; i++) {
+		const node = nodes[i];
+		if (node.className.includes("active")) all.push(node.textContent);
+	}
+	return all.join("");
+}
+
+function SetCategory(tag: string) {
```

Comment:
You should name this function better because all it's doing is returning an array, it doesn't set any data